lms/djangoapps/courseware/url_helpers.py

```diff
@@ -9,19 +9,23 @@
 from xmodule.modulestore.django import modulestore
 
 
-def get_redirect_url(course_key, usage_key):
+def get_redirect_url(course_key, usage_key, unified_course_view=False):
```

Comment:
Nit: My preference would be to not make this a keyword, but instead just have the temporary logic inside the method. I don't see why some callers would want to obey the Waffle flag, but others wouldn't.